opal/util/info.h

```diff
@@ -67,6 +67,7 @@ struct opal_info_entry_t {
     opal_cstring_t *ie_key;   /**< "key" part of the (key, value) pair */
     uint32_t ie_referenced;   /**< number of times this entry was internally
```

Comment:
As you removed the reference counting mechanism do we still need this field ?

Reply:
It's still needed, the implicit reference counting is still there (accesses through `opal_info_get/set`)